projects/cdk/observables/drag-and-drop-from.ts

```diff
@@ -35,6 +35,8 @@ export function tuiDragAndDropFrom(element: Element): Observable<TuiDragState> {
             tuiMouseDragFinishFrom(ownerDocument).pipe(
                 take(1),
                 map(event => new TuiDragState(TuiDragStage.End, event)),
+                // endWith(null) is not deprecated despite the notice,
+                // but IDE detects a wrong function signature, which is deprecated
```

Comment:
I am not sure that we need to add comment here.

This bug will be fixed in future, but this comment become some black-box unclear comment)

Reply:
fixed